packages/@aws-cdk-testing/framework-integ/test/aws-lambda/test/integ.lambda_with_dualStack_vpc.ts

```diff
@@ -0,0 +1,42 @@
+import { App, Stack } from 'aws-cdk-lib/core';
+import * as ec2 from 'aws-cdk-lib/aws-ec2';
+import { IntegTest } from '@aws-cdk/integ-tests-alpha';
+import * as lambda from 'aws-cdk-lib/aws-lambda';
+
+const app = new App();
+const stack = new Stack(app, 'DualStackVpc');
+
+const natProvider = ec2.NatProvider.gateway();
+const vpc = new ec2.Vpc(stack, 'DualStackProtocolVpc', {
+  ipProtocol: ec2.IpProtocol.DUAL_STACK,
+  subnetConfiguration: [
+    {
+      name: 'Pub1',
```

Comment:
Same as readme, let's make it `Public` or `Private1` so it's more clear to customers.

Reply:
Done